src/Orleans.Clustering.Consul/Options/ConsulGatewayListProviderOptions.cs

```diff
@@ -2,7 +2,7 @@
 using System.Collections.Generic;
 using System.Text;
 
-namespace OrleansConsulUtils.Options
+namespace Orleans.ConsulUtils.Options
```

Comment:
why is rootnamespace for this assembly Orleans.ConsulUtils, instead of Orleans.ConsulClustering?